mkdocs.yml

```diff
@@ -98,6 +98,18 @@ nav:
     - Configuration:
       - reference/configuration.md
       - reference/model_configuration.md
+#  - Tobiko Cloud: 
```

Comment:
Do we not need to expose this at the top level?

Reply:
Hey, not sure I follow your question, but here is that I think you were asking below:

While we build the docs, I have added the structure to the mkdocs.yml file and commented it out just for the time being. Trey had a brilliant idea of not exposing the menu option to many "empty" files. They will be there for us to add to and edit but not accessible from the top bar menu until they have useful content for the community. 
Of course, anyone can find them by directly putting the slug in, and be redirected to a page that says the content is coming soon, but most users don't navigate the docs like this!